rand_distr/src/binomial.rs

```diff
@@ -66,14 +64,19 @@ struct Binv {
     r: f64,
     s: f64,
     a: f64,
+    n: u64,
+    flipped: bool,
 }
 
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 struct Btpe {
     n: u64,
     p: f64,
-    // TODO consider precomputing more constants, there is a size/speed tradeoff to make here.
+    npq: f64,
+    p1: f64,
+    flipped: bool,
+    // This has the same size as `Binv` one could consider precomputing more.
```

Comment:
This comment is likely to cause confusion later (without the context of recent discussions on size reduction).